include/ogdf/basic/RegisteredArray.h

```diff
@@ -612,6 +612,11 @@ class RegisteredArrayWithDefault : public RegisteredArrayWithoutDefault<Registry
 	using RA = RegisteredArrayWithoutDefault<Registry, Value>;
 	Value m_default;
 
+	static_assert(std::is_copy_constructible_v<Value>,
+			"This RegisteredArrayWithDefault<Value> instantiation (e.g. NodeArray<Graph>) is "
+			"invalid because Value is not copy-constructible! "
+			"Use NodeArrayP<Graph> or NodeArray<unique_ptr<Graph>, false> instead.");
```

Comment:
Because we may actually be looking at an EdgeArray instead:
Use **e.g.** NodeArrayP<Graph> ...